models/repo.go

```diff
@@ -1444,120 +1444,6 @@ func RepoPath(userName, repoName string) string {
 	return filepath.Join(UserPath(userName), strings.ToLower(repoName)+".git")
 }
 
-// TransferOwnership transfers all corresponding setting from old user to new one.
```

Comment:
Should in case anyone is looking at this, it was moved to the newly created `repo_transfer.go` file https://github.com/go-gitea/gitea/pull/5744/files#diff-4a00b23928d7e63417b6a39f3052ec23R126